### DISCUSSION TITLE
Issue details

### DISCUSSION THREAD

**Discussion by nemanjaSokic** [mightymatth/arcli#7] [Ideas]
I really miss some details about an issue I want. What do you think about adding a new flag such as "-d" to show details like notes, and showing some attributes. Things about attributes could be a problem because most of them are custom.

There are APIs for notes (comments below an issue). In my company, sometimes these notes tell you more than issue descriptions.

Example API for notes: /issues/issue_id.json?include=journals

Details about attributes: https://www.redmine.org/projects/redmine/wiki/Rest_CustomFields 
For the beginning maybe is better to start with embedded issue fields?

**Comment by mightymatth** [comment]
Absolutely yes! If you have some time, you could add this details flag ("-d") to show additional notes. I'm not actively using Redmine, so you're more familiar with the missing features. You can freely discuss them here.

> For the beginning maybe is better to start with embedded issue fields?

What do you mean exactly with "embedded issue fields"? Can you further explain what would you do?

**Reply by nemanjaSokic** [reply]
Maybe , the better word is _native fields_.
I mean, fields witch comes from redmine itself. There is a posibilitiy to add a lot custom fields such as "Project type" (dropdown menu), "Client name" (input fields), but native fields comes with redmine instalation. Examples are "Assignee", "Priority",...
